sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/AzureConfiguration.java

```diff
@@ -9,5 +9,5 @@
 class AzureConfiguration {
     //TODO: Eventually remove these hardcoded strings with https://github.com/Azure/azure-sdk-for-java/issues/3141
     static final String NAME = "application-configuration";
-    static final String VERSION = "1.0.0-preview.4";
+    static final String VERSION = "1.0.0-preview.5";
```

Comment:
I already have a PR to do this.
https://github.com/Azure/azure-sdk-for-java/pull/5777/files#diff-0f18e2c84df27b7aac38e1364f384ef9L12